xbmc/pvr/windows/GUIEPGGridContainerModel.cpp

```diff
@@ -325,10 +326,10 @@ void CGUIEPGGridContainerModel::FindChannelAndBlockIndex(int channelUid, unsigne
 
 unsigned int CGUIEPGGridContainerModel::GetGridStartPadding() const
 {
-  unsigned int iEpgLingerTime = g_advancedSettings.m_iEpgLingerTime;
+  unsigned int iPastDays = CServiceBroker::GetSettings().GetInt(CSettings::SETTING_EPG_PAST_DAYSTODISPLAY) * 24 * 60;
```

Comment:
If this is `iPastDays`, why do you multiply by `24 * 60`?

Reply:
Good catch. Will change this. Thanks.